### PR TITLE
Create bundle, logging module

### PR DESCRIPTION
**Pull request by pbernad-leadin**
https://lead-in.monday.com/boards/72721864/pulses/1955163392

### CODE REVIEW

File: src/DependencyInjection/Facade.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Leadin\SurvivalKitBundle\DependencyInjection;
+
+use Psr\Container\ContainerInterface;
+
+abstract class Facade
+{
+    public const TAG        = 'survival_kit.facade';
```
Comment (leadinpier) [inline src/DependencyInjection/Facade.php:9]: seems not other choice, but public const are so ugly ...

**Review by leadinpier — APPROVED** [review]

File: src/Logging/LogContext.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * @method static self DEFAULT()
+ * @method static self MESSAGE()
+ * @method static self TRACKING()
+ * 
+ * Lists possible log contexts using private constants.
+ * Can be extended if the service requires additional contexts.
+ */
+class LogContext extends Enum
```
[the remark this answers is not in the document]
Comment (pbernad-leadin) [reply src/Logging/LogContext.php:15]: Just need to use an alias when extending this class, but not a big issue. I prefer it than naming here `BundleLogContext`

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+So there are a few rules:
+- No functional feature or code (no resource model)! Only technical code that can be widely reused
+- Tools provided must be independent as much as possible
+- Limit the number of external dependencies
+- Keep compatibility as much as possible and plan smooth migrations
+
+## This bundle provides the following:
+### Basic tools
+- logger tool
+- http client tool
```
[the remark this answers is not in the document]
Comment (pbernad-leadin) [reply README.md:18]: indeed not yet, but will be very soon

File: src/Logging/LogContext.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+{
+    protected const DEFAULT  = 'default';
+    protected const MESSAGE  = 'message';
```
[the remark this answers is not in the document]
Comment (pbernad-leadin) [reply src/Logging/LogContext.php:18]: to being able to use it in extending class

File: src/DependencyInjection/Facade.php (hunk 1 above)
Comment (pbernad-leadin) [reply src/DependencyInjection/Facade.php:9]: agree

File: src/Logging/LogContext.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (leadinpier) [reply src/Logging/LogContext.php:15]: I do not agree, using an alias in all our stacks instead of having BundleLogContext here seems more complicate

**Pull request merged**